item.rb

```diff
@@ -0,0 +1,7 @@
+class Item
+  attr_accessor :id
+
+  def initialize()
+    @id = 1
```

Comment:
As per the diagram, please include a custom setter method for all the other properties